packages/react/src/components/ImageGalleryModal/ImageGalleryModal.jsx

```diff
@@ -171,7 +171,7 @@ const ImageGalleryModal = ({
           secondaryButtonText={modalSecondaryButtonLabelText}
           modalHeading={deleteModalTitleText(selectedImage?.id)}
           size="xs"
-          iconDescription={modalCloseIconDescriptionText}
```

Comment:
iconDescription is being deprecated in favor of other attributes. On the Modal it's the aria-label.